### PR TITLE
(Feature) #2189 phase0 to phase1 upgrade popup

### PR DESCRIPTION
**Pull request by yaroslav-fedyshyn-nordwhale**
# Description

Add phase version config properties and /verify/phase endpoint which return the number of current phase version.

About https://app.zenhub.com/workspaces/gooddollar-5d50833888a83c6880d3e345/issues/gooddollar/gooddapp/2189

The link to the GoodDapp PR - https://github.com/GoodDollar/GoodDAPP/pull/2194

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -9,6 +9,10 @@ MNEMONIC=choice congress hobby buddy dutch busy army eager empty solution start
 INFURA_API=
 NETWORK=production
 LOG_LEVEL=trace
+# App release/phase version
+RELEASE_PHASE=
+PHASE_ZERO=
+PHASE_ONE=
```
Comment (serdiukov-o-nordwhale) [inline .env.example:15]:
```suggestion
```

it doesn't needed to show old env vars in the example - just the actual ones

File: src/server/server.config.js [hunk 2]
```diff
@@ -24,6 +24,12 @@ const conf = convict({
     default: version,
     env: 'VERSION'
   },
+  phase: {
+    doc: 'The application release/phase version',
+    format: Number,
+    default: -1,
```
Comment (serdiukov-o-nordwhale) [inline src/server/server.config.js:30]:
```suggestion
    default: 0,
```

0 is the default value on goddapp

File: src/server/server.config.js [hunk 3]
```diff
@@ -511,6 +517,20 @@ const mainNetworkId = get(ContractsAddress, `${network}-mainnet.networkId`, netw
 conf.set('ethereumMainnet', networks[mainNetworkId])
 conf.set('ethereum', networks[networkId])
 
+// phase options
+const phase = conf.get('phase')
```
Comment (serdiukov-o-nordwhale) [inline src/server/server.config.js:521]: aren't we had PHASE_ONE / PHASE_ZERO env vars before ? or just on dapp ?
if there were no PHASE_ONE / PHASE_ZERO on server before - this isn't needed, just add  RELEASE_PHASE with default value = 0

File: src/server/verification/verificationAPI.js [hunk 4]
```diff
@@ -855,6 +856,24 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
...
+   * @apiName Get Phase VErsion Number
+   * @apiGroup Verification
+   *
+   * @apiSuccess {Number} phase
+   * @apiSuccess {Boolean} success
+   * @ignore
+   */
+  app.get(
+    '/verify/phase',
+    passport.authenticate('jwt', { session: false }),
```
Comment (serdiukov-o-nordwhale) [inline src/server/verification/verificationAPI.js:871]: 1. we don't need authentication here
2. i think we don't need wrapAsync (and declare route function as async) as we don't have any async code or possible exceptions to be thrown
3. you don't use req and next so change function declaration to (_, res) => {
4. just to keep coverage level please add very simple unit tests which calls /verify/phase _without_ creds and checks response for { success: true, phase: phase value from config)

File: src/server/server.config.js (hunk 3 above)
Comment (yaroslav-fedyshyn-nordwhale) [reply src/server/server.config.js:521]: just on gooddap

Comment (serdiukov-o-nordwhale) [reply src/server/server.config.js:521]: okay so remove please all stuff added, just keep RELEASE_PHASE (default=0)

File: src/server/verification/__tests__/verificationAPI.js [hunk 5]
```diff
@@ -603,6 +603,14 @@ describe('verificationAPI', () => {
     })
   })
 
+  test('/verify/phase', async () => {
+    const res = await request(server).get('/verify/phase')
+    const { success, phase } = res.body || {}
+
+    expect(success).toBe(true)
+    expect(phase).toBe(0)
```
Comment (serdiukov-o-nordwhale) [inline src/server/verification/__tests__/verificationAPI.js:611]:
```suggestion
    const { phase } = Config
    
    await request(server)
      .get('/verify/phase')
      .expect(200, { success: true, phase })
```

File: src/server/verification/verificationAPI.js [hunk 6]
```diff
@@ -866,14 +866,10 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
...
-    passport.authenticate('jwt', { session: false }),
-    wrapAsync(async (req, res, next) => {
-      res.json({ success: true, phase: conf.phase })
-      res.end()
-    })
-  )
+  app.get('/verify/phase', (req, res, next) => {
+    res.json({ success: true, phase: conf.phase })
+    res.end()
+  })
```
Comment (serdiukov-o-nordwhale) [inline src/server/verification/verificationAPI.js:872]:
```suggestion
  app.get('/verify/phase', (_, res) => {
    const { phase } = conf
    
    res.json({ success: true, phase })
    res.end()
  })
```

**Pull request merged**